woocommerce/woocommerce-ios
# Add total_duration and page_number to orders_list_loaded Event

Part of #3654. 

I updated the `orders_list_event` to include `page_number` and the `total_duration`. This will be used to gather more information for the slow orders reports. 

https://github.com/woocommerce/woocommerce-ios/blob/10dbce2fb4269ea9db8496938202268190bd5179/WooCommerce/Classes/Analytics/WooAnalyticsEvent.swift#L113-L118

I also changed the custom property specifications of `WooAnalyticsEvent` so that it will allow other support property types instead of just `String`. 

https://github.com/woocommerce/woocommerce-ios/blob/10dbce2fb4269ea9db8496938202268190bd5179/WooCommerce/Classes/Analytics/WooAnalyticsEventPropertyType.swift#L3-L14

## Testing

1. Navigate to Orders.
2. Confirm that you will see a Tracks event log like this:
    ```
    Tracked orders_list_loaded, properties: [AnyHashable("page_number"): 1, AnyHashable("total_duration"): 1.2141919136047363, AnyHashable("blog_id"): 0, AnyHashable("status"): "processing", AnyHashable("is_wpcom_store"): false]
    ```
3. If you can, please also confirm in Tracks that the custom properties are logged. 

## Author Checklist
